flexinput/src/main/java/com/lytefast/flexinput/fragment/FlexInputFragment.java

```diff
@@ -126,6 +126,9 @@ public void onItemUnselected(final Object item) {
           }
 
           private void updateUi() {
+            if (getView() == null) {
```

Comment:
No guarantees on running on UI thread. Thus why we need to grab the view to post a runnable.